samples/durable-task-sdks/dotnet/Agents/PromptChaining/Worker/Services/IAgentService.cs

```diff
@@ -0,0 +1,443 @@
+using System.Text.Json;
+using Azure;
+using Azure.AI.Projects;
+using Azure.AI.Agents.Persistent;
+using Azure.Core;
+using Azure.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System.Text;
+
+namespace AgentChainingSample.Services;
+
+/// <summary>
+/// Interface for agent services
+/// </summary>
+public interface IAgentService
```

Comment:
Is this interface needed?  The users of the services derived from the base type that implements this interface to not seem to use it.  (Activities are not swapping one agent service for another, for example; each activity is using a specific agent service.)